src/gba/memory.c

```diff
@@ -383,6 +385,8 @@ static void GBASetActiveRegion(struct ARMCore* cpu, uint32_t address) {
 		LOAD_32(value, address & (SIZE_CART0 - 4), memory->rom); \
 	} else if (memory->mirroring && (address & memory->romMask) < memory->romSize) { \
 		LOAD_32(value, address & memory->romMask, memory->rom); \
+	} else if (memory->vfame.cartType && address < BASE_CART1) { \
```

Comment:
Ah I see. Makes sense! Sorry this is the first time I've worked with the GBA at all, forgive me if I ask stupid questions sometimes